### PR TITLE
docs: GettingStarted Simplified

### PR DESCRIPTION
**Pull request by mudassir-agoric**
I am changing documentation pages for the Getting Started dapp. Here is a summary:

1. The headings/steps on the index page look cluttered (a sentiment shared by multiple new devs), so I have numbered them as a first step.
2. After the `start-contract` command step, there is an explanation of what is happening under the hood. This seems unnecessary; therefore, I am hiding it in `<details>` tag. 
3. In the Keplr installation, there is a missing step regarding `selecting chains` (or rather not selecting them) that confuses some people, so I am adding it.
4. `print-key` has been removed from the code but there are still some references in the docs so I am removing them.

### CODE REVIEW

File: main/guides/getting-started/explainer-deploying-a-smart-contact.md [hunk 1]
```diff
@@ -37,15 +36,15 @@ Note the calling the `yarn start:contract` command is the same as running `yarn
     "docker:bash": "docker compose exec agd bash",
     "docker:make": "docker compose exec agd make -C /workspace/contract",
     "make:help": "make list",
-    "start": "yarn docker:make clean start-contract print-key",
+    "start": "yarn docker:make clean start-contract",
```
Comment (dckc) [inline main/guides/getting-started/explainer-deploying-a-smart-contact.md:39]: :+1:

File: main/guides/getting-started/index.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ Before getting started, there are some resources you might want to keep handy in
 
 Currently Agoric supports macOS and Linux (including [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/about)). This tutorial is based on an installation of [Ubuntu 22.04 LTS](https://ubuntu.com/download/desktop). If you're using a different operating system, some variation may be required.
 
-## Installing Prerequisites
+## 1. Installing Prerequisites
```
Comment (dckc) [inline main/guides/getting-started/index.md:35]: We used to have tools to detect broken links to section headings, but we don't any more.

 - https://github.com/Agoric/documentation/issues/1011

I'm not comfortable changing heading text until we can do that sort of checking.

**Review by dckc — COMMENTED** [review]
I'm not sure about changing heading text...

File: main/guides/getting-started/index.md (hunk 2 above)
Comment (kbennett2000) [reply main/guides/getting-started/index.md:35]: Agreed - we should probably delay changing heading text until we get some tooling in place...

Comment (mudassir-agoric) [reply main/guides/getting-started/index.md]: Agreed. I did not know headings were linked. I have removed the changes to headings in a new commit.

**Review by kbennett2000 — APPROVED** [review]
Looks good to me

**Comment by mudassir-agoric** [comment]
> re failed cloudflare build, see [#1077 (review)](https://github.com/Agoric/documentation/pull/1077#pullrequestreview-2044615508)

Looks like @ansabgillani has fixed this in #1091 - so I guess we will wait for him to merge that.

**Comment by mudassir-agoric** [comment]
> re failed cloudflare build, see [#1077 (review)](https://github.com/Agoric/documentation/pull/1077#pullrequestreview-2044615508)

I guess it is fixed now - so I am merging!

**Pull request merged**